.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+-   repo: local
+    hooks:
+    -   id: mix-test
+        name: 'elixir: mixing tests'
+        entry: mix test
```

Comment:
True, i need to update the readme with instructions for installing `pre-commit`

installing with brew was the quickest way for me to get up and running.

Found [this pre_commit_hook] hex module but seems like its under construction. I haven't given it a go but could be a valid alternative. Here is it's github: https://github.com/tyrchen/ex_pre_commit_hook